File: Shareen.Domain/Lobby.cs
namespace Shareen.Domain;

public class Lobby
{
    public Guid Id { get; set; }
    public Guid ChatId { get; set; }
    public Chat Chat { get; set; }
    public List<User> Users { get; set; }
    public string Name { get; set; }
    public DateTime TimeCreated { get; set; }
    public string UniqueLink { get; set; }
}

File: src/internal/domain/lobby.go
package domain

// Lobby describes lobby.

type Lobby struct {
	ID       int64     `json:"id,omitempty"`
	LobbyURL string    `json:"lobby_url"`
	VideoURL string    `json:"video_url"`
	Timing   string    `json:"timing"`
	Pause    bool      `json:"pause"`
	Users    []*User   `json:"users"`
	Chat     []Message `json:"chat"`
}
